package.json

```diff
@@ -29,7 +29,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "node scripts/pre-build.js && react-scripts build",
-    "test": "yarn prettier:fix && react-scripts test && yarn test:e2e:run",
+    "test": "yarn prettier:fix && react-scripts test",
```

Comment:
Why?

Reply:
I somehow convinced myself that this was causing an issue, but can no longer reproduce it.  So, I put it back.  Thanks for making me think it through!

Reply:
It does locally because `react-scripts test` runs in watch mode by default so when you stop the watch the e2e tests start to run, but in ci it's not a problem.